.github/workflows/main.yml

```diff
@@ -0,0 +1,197 @@
+name: Build
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  IMAGE: secure-message-v2
+  REGISTRY_HOSTNAME: eu.gcr.io
+  HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
+  RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
+  CHART_DIRECTORY: _infra/helm/secure-message-v2
+  SPINNAKER_TOPIC: ${{ secrets.SPINNAKER_TOPIC }}
+  ARTIFACT_BUCKET: ${{ secrets.ARTIFACT_BUCKET }}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build & package"
+  build:
+    name: Build & Package
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres:9.4
```

Comment:
Can you change this to 9.6, it's what we use normally so we should be at least matching instead of behind.

Reply:
Fixed.